mlrun/model.py

```diff
@@ -1284,7 +1284,7 @@ def ui_url(self) -> str:
         """UI URL (for relevant runtimes)"""
         self.refresh()
         if not self._status.ui_url:
-            print("UI currently not available (status={})".format(self._status.state))
+            print(f"UI currently not available (status={self._status.state})")
```

Comment:
A print in the code... should be a log.
Can be caught by the `T201` rule:
https://docs.astral.sh/ruff/rules/#flake8-print-t20